pages/leverage-metadata-collections.mdx

```diff
@@ -0,0 +1,160 @@
+import { Steps } from "nextra/components";
+import { Callout } from "nextra/components";
+import { Box, Text, Section } from "@radix-ui/themes";
+
+# Leverage Metadata Collections
+
+<Section size="1">
+  <Text size="4">
+    Canopy generates new IIIF Collections based on the metadata labels included in your `config/canopy.json` file.
+    For each unique string value associated with the label across the IIIF manifests found in your project, a new
+    IIIF Collection is created. These Collections are used to create facets across your site and are also available
+    for reuse in custom content.
+  </Text>
+</Section>
+
+<Callout>
+  This guide assumes you have a Canopy IIIF project. See the [Create a
+  Project](/get-started) guide to get started.
+</Callout>
+
+## Use Case
+
+You are a researcher creating a digital scholarship site and would like to create a custom content page
+with a Slider component that displays all the works in your collection that have a "Subject" of "War pensions".
+
```

Comment:
Maybe unnecessary, but what if there was a quick little snippet of `json` here illustrating where "Subject" sits within the example Manifest's `metadata`?